examples/routing_service/routing_service_constrained_wan/wandemo/server.tcp.transport.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
+<!--
+ (c) 2005-2020 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+ RTI grants Licensee a license to use, modify, compile, and create derivative
+ works of the Software.  Licensee has the right to distribute object form only
+ for use with RTI products.  The Software is provided "as is", with no warranty
+ of any type, including any warranty for fitness for any purpose. RTI is under
+ no obligation to maintain or support the Software.  RTI shall not be liable for
+ any incidental or consequential damages arising out of the use or inability to
+ use the software.
+ -->
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+     xsi:noNamespaceSchemaLocation="https://community.rti.com/schema/6.0.0/rti_routing_service.xsd">
+    <!-- RTI Routing service examples for TCP over WAN                          -->
+    <!-- Note: Change the IPs and ports below to fit yours                      -->
+    <!-- ********************************************************************** -->
+    <!-- TCP (domain 1) => UDP/Shared memory (domain 2)                         -->
+    <!-- ********************************************************************** -->
+    <routing_service name="TCP_2">
+        <annotation><documentation>In a WAN, routes from TCP to UDP or shared memory. Remember to edit the file and change the public IP address.</documentation></annotation>
```

Comment:
try to shorten this line

Reply:
should be fixed, it is now shorter than the comment from the original template